### ISSUE TITLE
Add more randomized SVD algorithms

### ISSUE THREAD

**Issue by sdrave** [pymor/pymor#296]
Randomized SVD algorithms work perfectly with matrix-free operators, so they would fit perfectly into pyMOR. For a start, we could take a look at Facebook's [implementation](https://github.com/facebook/fbpca).

**Comment by sdrave** [comment]
Would be very nice to have something in the next release. Maybe @deneick wants to take a shot after #544 has been implemented.

**Comment by sdrave** [comment]
With #665, we have at least a randomized range finder algorithm in pyMOR. I guess this is enough to change the milestone ..

**Comment by artpelling** [comment]
Closed by #1552.
